src/components/Notification/Notification.tsx

```diff
@@ -64,15 +71,41 @@ export { logger as notificationLogger };
  */
 export const Notification = ({
   type,
-  action,
+  actions,
   header,
   description,
   isElevated = false,
+  isInline = true,
+  occurences = 0,
+  locale = "en-US",
```

Comment:
I decided that it would be easier to allow the user to pass in the `locale` prop as necessary, rather than use the `navigator` object provided by browsers, as this will also allow for the Component to be rendered server-side where client-side objects might not be available.

Reply:
unless you _can't_ use `navigator`, I'd set the default to be:
```suggestion
  locale = navigator?.languages[0] ?? "en-US",
```